FreeRTOS/Demo/RISC-V_RV32_SiFive_HiFive1-RevB_IAR/FreeRTOSConfig.h

```diff
@@ -94,7 +94,7 @@
 
 /* Normal assert() semantics without relying on the provision of an assert.h
  * header file. */
-#define configASSERT( x )                   if( ( x ) == 0 ) { taskDISABLE_INTERRUPTS(); __asm volatile( "ebreak" ); for( ;; ); }
+#define configASSERT( x )                   if( ( x ) == 0 ) { taskDISABLE_INTERRUPTS(); for( ;; ); }

```

Comment:
I understand why the `ebreak` statement was removed from the loop. However, should we not keep it here? Just in case an assert is hit, the debugger will stop showing the bug before we enter the loop.
Or am I missing something?